lib/committee/middleware/response_validation.rb

```diff
@@ -5,6 +5,8 @@ def initialize(app, options={})
       @validate_errors = options[:validate_errors]
     end
 
+    attr_reader :validate_errors
```

Comment:
Would you mind just sticking this at the top of the class (above the constructor)?